LayoutTests/inspector/timeline/timeline-event-screenshots.html

```diff
@@ -28,11 +28,22 @@
             let recording = WI.timelineManager.activeRecording;
             let screenshotTimeline = recording.timelines.get(WI.TimelineRecord.Type.Screenshots);
             let records = screenshotTimeline.records;
-            InspectorTest.expectGreaterThan(records.length, 0, "Should have at least 1 Screenshot record.");
+            let oldLength = records.length;
+            InspectorTest.expectGreaterThan(oldLength, 0, "Should have at least 1 Screenshot record.");
 
+            InspectorTest.assert(records[0].startTime, "Screenshot record should have a timestamp.");
             InspectorTest.expectGreaterThan(records[0].imageData.length, 0, "Screenshot record should contain image data.");
-            InspectorTest.expectGreaterThan(records[0].width, 0, "Screenshot record width should be non-zero.");
-            InspectorTest.expectGreaterThan(records[0].height, 0, "Screenshot record height should be non-zero.");
+
+            InspectorTest.log("Adding screenshot from console...");
+            await InspectorTest.evaluateInPage(`console.screenshot()`);
+
+            InspectorTest.expectGreaterThan(records.length, oldLength, "Should have added another Screenshot record.");
```

Comment:
Nit: Since capturing a screenshot should only result in a single new record, should we actually expect `records.length === oldLength + 1`?